public/style.css

```diff
@@ -0,0 +1,38 @@
+body {
```

Comment:
https://www.w3schools.com/css/css_intro.asp  know you said you'd like a tutorial on CSS, i found this super helpful (although I wouldn't claim to be an  expert but this helped massively)

Reply:
saved bookmark